d2/src/main/java/com/linkedin/d2/xds/XdsToD2PropertiesAdaptor.java

```diff
@@ -271,8 +279,10 @@ public void onChanged(XdsClient.NodeUpdate update)
             }
             publishClusterData(clusterName, clusterProperties);
           }
-          catch (PropertySerializationException e)
+          catch (Exception e)
           {
+            _clusterEventBus.publishInitialize(clusterName, null);
+            // still notify event bus to avoid timeout in case some subscribers are waiting for the data
```

Comment:
```suggestion
            // still notify event bus to avoid timeout in case some subscribers are waiting for the data
            _clusterEventBus.publishInitialize(clusterName, null);
```